tobac/tests/test_feature_detection.py

```diff
@@ -0,0 +1,29 @@
+import tobac.testing
+import tobac.feature_detection as feat_detect
+
+
+
+
+def test_feature_detection_multithreshold_timestep():
+    '''
+    Tests ```tobac.feature_detection.feature_detection_multithreshold_timestep
+    '''
+    import numpy as np
+    from tobac import testing
+    from tobac import feature_detection 
+    # start by building a simple dataset with a single feature and seeing 
+    # if we identify it
+    
+    test_data = np.zeros((50,50))
+    test_data = testing.make_feature_blob(test_data, 20, 20, h1_size=5, h2_size=5, amplitude=2)
```

Comment:
Might be good to set the feature location as a variable to make it clearer, and link the test value of the assertion to this. Also is it possible to set it up so that the calculated feature locations is the same as the test data location to make this clearer?

Reply:
Yeah, this is a good point and has caused a headache. I've switched it from rounding the `start_h1` etc. to consistently running `np.ceil` in `tobac.testing.make_feature_blob` to resolve this.